packages/components/organisms/f-offers/package.json

```diff
@@ -58,6 +59,8 @@
     "@vue/test-utils": "1.0.3",
     "node-sass-magic-importer": "5.3.2",
     "js-cookie": "2.2.1",
-    "@justeat/f-trak": "0.7.1"
+    "@justeat/f-trak": "0.7.1",
+    "@justeat/f-media-element": "0.3.0",
+    "@justeat/f-button": "1.5.0"
```

Comment:
Any reason not to use the latest `f-button` - `1.8.0` at time of writing?

Reply:
No sorry, its what my webstorm suggested, will make the change

Reply:
1.7.0 is only option